lib/Models/Leaflet.ts

```diff
@@ -110,7 +110,7 @@ export default class Leaflet extends GlobeOrMap {
       maxZoom: 14, //this.maximumLeafletZoomLevel,
       zoomSnap: 1, // Change to  0.2 for incremental zoom when Chrome fixes canvas scaling gaps
       preferCanvas: true,
-      worldCopyJump: true
+      worldCopyJump: false
```

Comment:
Intended flip here?

Reply:
It comes from https://github.com/TerriaJS/terriajs/pull/3679